custom_components/securitas/alarm_control_panel.py

```diff
@@ -226,9 +217,10 @@ def update_status_alarm(self, status: CheckAlarmStatus = None):
             elif status.protomResponse == "P":
                 self._state = STATE_ALARM_ARMED_HOME
             elif (
-                status.protomResponse == "E"
-                or status.protomResponse == "B"
-                or status.protomResponse == "C"
+                status.protomResponse == "E" # PERI
+                or status.protomResponse == "B" # PERI + ARMED_HOME
+                or status.protomResponse == "C" # PERI + ARMED_NIGHT
+                or status.protomResponse == "A" # PERI + ARMED_AWAY
```

Comment:
Adding this new `protomResponse` and a text explanation of what each means.

Reply:
Fantastic!